docs/getting_started/notifications.md

```diff
@@ -25,4 +27,36 @@ As a result, you should see the following logs on the server side
 ```
 07:45:02.684 [info] New incoming server WebSocket connection, accepting
 07:45:02.688 [info] Server WS authenticated.
+```
+
+To receive server notifications through webhooks you have to pass `webhook_url` during creating room.
+Example using Elixir Server SDK:
+```elixir
+server_address = "localhost:5002"
+server_api_token = "development"
+webhook_url = "http://localhost:5003/"
+
+client = Jellyfish.Client.new(server_address: server_address, server_api_token: server_api_token)
+{:ok, %Jellyfish.Room{id: room_id}, jellyfish_address} = Jellyfish.Room.create(client, max_peers: 10, webhook_url: webhook_url)
+```
+
+To decode received notification through webhook, you can use a function from SDK.
+Example using Elixir Server SDK:
```

Comment:
We lack the information that user has to receive post request on its own and pass its body to our SDK function that will decode it

Reply:
IMO, it's evident that the user has to receive post requests on its own because it is how webhooks work. As for passing the body to our SDK, I am not sure what other can be described here but I will try to change it.